### ISSUE TITLE
CI failure on release

### ISSUE THREAD

**Issue by obycode** [hirosystems/clarity-lsp#64]
https://github.com/hirosystems/clarity-lsp/runs/4885917623?check_suite_focus=true

```
[5:24:58 PM] [semantic-release] › ✖  An error occurred while running semantic-release: Error: Command failed with exit code 128: git tag v0.6.0 7f0e1810de38a5eff151094a35aaea21928fcbfe
249
Error: Error: Command failed with exit code 128: git tag v0.6.0 7f0e1810de38a5eff151094a35aaea21928fcbfe
250
fatal: tag 'v0.6.0' already exists
```

It's unclear at first glance why the version number was not incremented to 0.6.1.

**Comment by obycode** [comment]
Attempting to bump the version with #67.

**Comment by obycode** [comment]
I attempted to release another version, hoping it would bump the version number and avoid this problem, but it still failed. https://github.com/hirosystems/clarity-lsp/actions/runs/1752154316

**Comment by obycode** [comment]
It finally worked!
![image](https://user-images.githubusercontent.com/1473715/151247565-f9896802-d6f0-45ba-b428-4df8d0065cca.png)
The "Preview" tag is still there for some reason, but at least the new version is released.

What had happened was that the tag `v0.6.0` was pointing to a commit that was not on the main branch, so the release tool was not finding it. This then caused a conflict when it tried to do another release with the same name. To fix it, I manually relocated the `v0.6.0` branch and force pushed it to the server, then made two new PRs.
